resolversrv/tests/unittests/test_db_query.py

```diff
@@ -0,0 +1,304 @@
+import sys
+import os
+
+PROJECT_HOME = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../'))
+sys.path.append(PROJECT_HOME)
+
+from flask_testing import TestCase
+import unittest
+
+import psycopg2
+import testing.postgresql
+import json
+
+import resolversrv.app as app
+from resolversrv.views import LinkRequest
+from resolversrv.models import DataLinks
+
+class test_resolver(TestCase):
+    """tests for generation of resolver"""
+
+
+    # Reference to testing.postgresql database instance
+    db = None
+
+    # Connection to the database used to set the database state before running each
+    # test
+    db_con = None
+
+    # Map of database connection parameters passed to the functions we're testing
+    db_conf = None
+
+    current_app = None
+
+    query_link_type = "SELECT * FROM public.datalinks WHERE bibcode = '{bibcode}' AND (link_type = '{link_type}')"
```

Comment:
i see now, the __table_args__ is a 'namespace'; this is only used by data pipeline and will be phased out too; microservices won't need it

also, we should use ORM whenever it is possible, so:

```
session.query(DataRecord).filter_by(or_(DataRecord.link_type==type, DataRecord.link_sub_type==subtype))
```

Reply:
Is this for reinforcement, since to me, the line of code you are showing me here, is ORM query.